spark/src/main/scala/io/substrait/spark/logical/ToLogicalPlan.scala

```diff
@@ -331,6 +334,34 @@ class ToLogicalPlan(spark: SparkSession) extends DefaultRelVisitor[LogicalPlan]
   override def visit(localFiles: LocalFiles): LogicalPlan = {
     val schema = ToSubstraitType.toStructType(localFiles.getInitialSchema)
     val output = schema.map(f => AttributeReference(f.name, f.dataType, f.nullable, f.metadata)())
+
+    // spark requires that all files have the same format
+    val formats = localFiles.getItems.asScala.map(i => i.getFileFormat.orElse(null)).distinct
+    if (formats.length != 1) {
+      throw new UnsupportedOperationException(s"All files must have the same format")
+    }
+    val (format, options) = formats.head match {
+      case csv: FileFormat.DelimiterSeparatedTextReadOptions =>
+        val opts = scala.collection.mutable.Map[String, String](
+          "delimiter" -> csv.getFieldDelimiter,
+          "quote" -> csv.getQuote,
+          "header" -> (csv.getHeaderLinesToSkip match {
+            case 0 => "false"
+            case 1 => "true"
+            case _ =>
+              throw new UnsupportedOperationException(
+                s"Cannot configure CSV reader to skip ${csv.getHeaderLinesToSkip} rows")
+          }),
+          "escape" -> csv.getEscape,
+          "maxColumns" -> csv.getMaxLineSize.toString
```

Comment:
this sounds wrong, maxColumns in spark is for _columns_ while maxLineSize in Substrait is for _bytes_

Reply:
Good catch.  There doesn't seem to be a Spark equivalent option for this, so I'll just remove the mapping and ignore.